docassemble/MotionToStayEviction/data/questions/review_screen.yml

```diff
@@ -0,0 +1,386 @@
+---
+event: review_case_info
+id: |-
+  review screen
+question: |-
+  Review your answers
+review:
+  - Edit: users.revisit
+    button: |
+      **Your party: (Edit to change name and address)**
+
+      %  for my_var in users:
+        * ${ my_var.name }
+      % endfor
+  - Edit: other_parties.revisit
```

Comment:
Editing other parties doesn't seem to work, I think it's probably because the `revisit` attribute isn't already set, which is because we aren't gathering the parties the normal way. I think we'll need to put a patch for that in `add_existing_users`?